### PR TITLE
restore: remove restore meta sql in create table

### PR DESCRIPTION
**Pull request by YuJuncen**
### What problem does this PR solve?

Speed up create tables via omit a DDL job.

Problem Summary:

### What is changed and how it works?

What's Changed: Remove all DDLs of rebasing autos.

How it Works: When creating table with `CreateTableWithInfo`, TiDB has rebased the auto ids via the table info:

https://github.com/pingcap/tidb/blob/a9cba7c8e8b6da512609863b56ee7265c06c50dc/ddl/ddl_api.go#L1924-L1937

It's no meaning for rebasing the auto ids again.

### Check List 

Tests 

- [ ] Integration test (at `br_views_and_sequences`)

### Release note 

```release-note
Merged the rebase auto id operation into create table.
```

### CODE REVIEW

File: br/tests/br_views_and_sequences/run.sh [hunk 1]
```diff
@@ -29,6 +33,13 @@ run_sql "create view $DB.view_3 as select m from $DB.table_1 union select a * b
 run_sql "drop view $DB.view_1;"
 run_sql "create view $DB.view_1 as select 133 as m;"
 
+run_sql "create table $DB.auto_inc (n int primary key AUTO_INCREMENT);"
+run_sql "insert into $DB.auto_inc values (), (), (), (), ();"
+last_id=$(run_sql "select n from $DB.auto_inc order by n desc limit 1" | trim_sql_result)
+
+run_sql "create table $DB.auto_rnd (n BIGINT primary key AUTO_RANDOM(8));"
+last_rnd_id=$(run_sql "insert into $DB.auto_rnd values (), (), (), (), ();select last_insert_id();" | trim_sql_result )
```
Comment (kennytm) [inline br/tests/br_views_and_sequences/run.sh:41]:
```suggestion
last_rnd_id=$(run_sql "insert into $DB.auto_rnd values (), (), (), (), ();select last_insert_id() & 0xffffffffffffff;" | trim_sql_result )
```

File: br/tests/br_views_and_sequences/run.sh [hunk 2]
```diff
@@ -39,11 +50,20 @@ run_br restore db --db $DB -s "local://$TEST_DIR/$DB" --pd $PD_ADDR
...
 run_sql "insert into $DB.table_2 (c) values (33);"
 seq_val=$(run_sql "select a >= 8 and b >= 4 as g from $DB.table_2 where c = 33;" | tail -1)
 [ "$seq_val" = 'g: 1' ]
 
-run_sql "drop schema $DB"
+run_sql "insert into $DB.auto_inc values ();"
+last_id_after_restore=$(run_sql "select n from $DB.auto_inc order by n desc limit 1;" | trim_sql_result)
+[ $last_id_after_restore -gt $last_id ]
+rnd_last_id_after_restore=$(run_sql "insert into $DB.auto_rnd values ();select last_insert_id();" | trim_sql_result )
+[ $(( rnd_last_id_after_restore & 0xff )) -gt $(( last_rnd_id & 0xff )) ]
```
Comment (kennytm) [inline br/tests/br_views_and_sequences/run.sh:64]:
```suggestion
rnd_last_id_after_restore=$(run_sql "insert into $DB.auto_rnd values ();select last_insert_id() & 0xffffffffffffff;" | trim_sql_result )
[ $rnd_last_id_after_restore -gt $last_rnd_id ]
```

`auto_random(8)` means the upper 8 bits are randomized, not that the lower 8 bits are used as counter.

**Review by kennytm — COMMENTED** [review]
rest LGTM

**Review by kennytm — APPROVED** [review]

**Review by 3pointer — APPROVED** [review]

**Comment by kennytm** [comment]
/run-check_dev_2

```
[2021-08-30T06:12:03.525Z] The following test cases take too long to finish:

[2021-08-30T06:12:03.525Z] --- PASS: TestIndexRangeForUnsignedAndOverflow (5.09s)

[2021-08-30T06:12:03.525Z] --- PASS: TestIndexRangeEliminatedProjection (5.21s)

[2021-08-30T06:12:03.525Z] --- PASS: TestPrefixIndexRangeScan (5.21s)

[2021-08-30T06:12:03.526Z] --- PASS: TestTableRange (5.21s)

[2021-08-30T06:12:03.526Z] --- PASS: TestCompIndexMultiColDNF1 (5.21s)

[2021-08-30T06:12:03.526Z] --- PASS: TestPrefixIndexMultiColDNF (5.21s)

[2021-08-30T06:12:03.526Z] --- PASS: TestIndexRangeForBit (5.21s)

[2021-08-30T06:12:03.526Z] --- PASS: TestPrefixIndexAppendPointRanges (5.21s)

[2021-08-30T06:12:03.526Z] --- PASS: TestCompIndexDNFMatch (5.21s)

[2021-08-30T06:12:03.526Z] --- PASS: TestIndexStringIsTrueRange (5.21s)

[2021-08-30T06:12:03.526Z] --- PASS: TestIndexRangeForYear (5.21s)

[2021-08-30T06:12:03.526Z] --- PASS: TestIndexRangeForDecimal (5.21s)

[2021-08-30T06:12:03.526Z] --- PASS: TestIndexRange (5.21s)

[2021-08-30T06:12:03.526Z] --- PASS: TestCompIndexMultiColDNF2 (6.34s)
```

**Pull request merged**